### PR TITLE
ElasticSearch upgrade

### PR DESCRIPTION
**Pull request by cruffinoni**
ElasticSearch (ES) has been upgraded to 6.8 on staging and will be upgrade to 6.7 on production. Along to this upgrade, the ES library for the API has been updated and older versions has been removed from the `vendor` (for govendor) file.
The tool `go fmt` has been used in almost every folder because of the huge amount of edited files due to the ES library upgrade so, there is a lot of little changes accross the repository.
After a short discussion with @hug33k, there was a mistake in the `README.md` file and has been corrected.

### CODE REVIEW

File: es/billingRepository.go [hunk 1]
```diff
@@ -17,7 +17,8 @@ package es
 import (
 	"context"
 
-	"gopkg.in/olivere/elastic.v5"
+	//"github.com/olivere/elastic"
```
Comment (giubil) [inline es/billingRepository.go]: You have a comment that shouldn't be here

File: es/client.go [hunk 2]
```diff
@@ -24,7 +24,8 @@ import (
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/credentials/ec2rolecreds"
 	"github.com/trackit/jsonlog"
-	"gopkg.in/olivere/elastic.v5"
+	//"github.com/olivere/elastic"
```
Comment (giubil) [inline es/client.go]: Dito

File: vendor/vendor.json [hunk 3]
```diff
@@ -503,6 +503,39 @@
...
+			"revisionTime": "2019-06-30T15:32:01Z",
+			"version": "v6",
+			"versionExact": "v6.2.21"
+		},
+		{
+			"checksumSHA1": "AmcqvKDAUGvVsrf8NDCQ/B8cMj4=",
+			"path": "github.com/olivere/elastic/uritemplates",
+			"revision": "89a71b89268052a9ba7a8bd9012fa87bf086c1fd",
+			"revisionTime": "2019-06-30T11:54:38Z"
+>>>>>>> cfa4129... ES API Library upgrade to 6.2.21 along with the ES 6.* upgrade + Removed obsolete ES API library + Usage of go fmt almost every folder due to huge amount of files edited.
```
Comment (giubil) [inline vendor/vendor.json]: Forgot to remove a merge conflict comment

File: vendor/vendor.json [hunk 4]
```diff
@@ -619,5 +634,5 @@
 			"revisionTime": "2017-09-19T05:20:08Z"
 		}
 	],
-	"rootPath": "github.com/trackit/trackit-server"
+	"rootPath": "github.com/trackit/trackit"
```
Comment (giubil) [inline vendor/vendor.json:616]: If this is changed only in this PR, does it mean that if we deploy stg right now it will be broken ?

**Review by giubil — CHANGES_REQUESTED** [review]
LGTM, except for a few comments left

File: vendor/vendor.json (hunk 4 above)
Comment (giubil) [reply vendor/vendor.json:616]: Nevermind, this fix was already merged in `stg`

File: vendor/vendor.json [hunk 5]
```diff
@@ -555,12 +540,6 @@
 			"revision": "9088e4c7b34b3174ae879fc975deb18ac62ac8d3",
 			"revisionTime": "2017-05-14T20:27:02Z"
 		},
-		{
```
Comment (giubil) [inline vendor/vendor.json:525]: Is this unused ? Why was this removed ?

**Review by giubil — CHANGES_REQUESTED** [review]

File: vendor/vendor.json (hunk 5 above)
Comment (giubil) [reply vendor/vendor.json:525]: Ah we changed libraries, nevermind

**Review by giubil — APPROVED** [review]

File: aws/usageReports/ec2/monthlyReport.go [hunk 6]
```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/trackit/jsonlog"
-	"gopkg.in/olivere/elastic.v5"
+	"github.com/olivere/elastic"
```
Comment (hug33k) [inline aws/usageReports/ec2/monthlyReport.go:28]: Wrong import order

File: aws/usageReports/history/history.go [hunk 7]
```diff
@@ -23,7 +23,7 @@ import (
 	"time"
 
 	"github.com/trackit/jsonlog"
-	"gopkg.in/olivere/elastic.v5"
+	"github.com/olivere/elastic"
```
Comment (hug33k) [inline aws/usageReports/history/history.go:25]: Wrong import order

File: es/client.go [hunk 8]
```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/credentials/ec2rolecreds"
 	"github.com/trackit/jsonlog"
-	"gopkg.in/olivere/elastic.v5"
+	"github.com/olivere/elastic"
```
Comment (hug33k) [inline es/client.go:26]: Wrong import order

File: es/simplifyCostsDocument.go [hunk 9]
```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/trackit/jsonlog"
 
-	"gopkg.in/olivere/elastic.v5"
+	"github.com/olivere/elastic"
```
Comment (hug33k) [inline es/simplifyCostsDocument.go:24]: Wrong import order

**Review by hug33k — CHANGES_REQUESTED** [review]

**Review by hug33k — APPROVED** [review]

**Pull request merged**